python/semantic_kernel/models/ai/chat_completion/chat_history.py

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from typing import Iterator, List, Optional
+
+from pydantic import Field
+
+from semantic_kernel.contents.chat_message_content import ChatMessageContent
+from semantic_kernel.kernel_pydantic import KernelBaseModel
+from semantic_kernel.models.ai.chat_completion.chat_role import ChatRole
+
+
+class ChatHistory(KernelBaseModel):
+    """
+    This class holds the history of chat messages from a chat conversation.
+
+    Note: the constructor takes a system_message parameter, which is not part
+    of the class definition. This is to allow the system_message to be passed in
+    as a keyword argument, but not be part of the class definition.
+
+    Attributes:
+        messages (List[ChatMessageContent]): The list of chat messages in the history.
```

Comment:
How can one add "AzureChatMessageContent" messages?

Reply:
One can add a specific chat message type to the chat history object they're working with. This will become more apparent when we begin to handle automatic function calling and we add tool call messages to a history.

Reply:
I see the add_message uses params, so you would have to manually add a message to `history.messages.append(msg)`, maybe make a add_message variant where you can just add a ChatMessageContent object?